cyber/common/global_data.h

```diff
@@ -58,6 +58,12 @@ class GlobalData {
   void EnableSimulationMode();
   void DisableSimulationMode();
 
+  void EnableOctopusMode();
+  bool IsOctopusMode() const;
+
+  void SetADCId(const int& adc_id);
+  int ADCId() const;
```

Comment:
cyber global data is a singleton, if we set the adc_id here, it cannot be set to different values for different planning modules.